orkaudio/filters/LiveStream/LiveStreamFilter.cpp

```diff
@@ -0,0 +1,265 @@
+/*
+ * Oreka -- A media capture and retrieval platform
+ *
+ */
+#pragma warning(disable : 4786) // disables truncated symbols in browse-info warning
+
+#define _WINSOCKAPI_ // prevents the inclusion of winsock.h
+
+#include "LiveStreamFilter.h"
+#include <log4cxx/logger.h>
+#include "AudioCapture.h"
+#include <iostream>
+#include <string>
+#include <cstring>
+#include "Utils.h"
+#include "srs_librtmp.h"
+#include <queue>
+#include "ConfigManager.h"
+
+#define BUFFER_SAMPLES 8000
+
+static log4cxx::LoggerPtr s_log = log4cxx::Logger::getLogger("plugin.livestream");
+
+template <class T>
+std::string toString(const T &value)
+{
+	std::ostringstream os;
+	os << value;
+	return os.str();
+}
```

Comment:
this function is not used/duplicate. You might want to remove this.

Reply:
removed